File: my-code/2619-categorize-box-according-to-criteria/solution.py
class Solution(object):
    def categorizeBox(self, l, w, h, m):
        v=l*w*h
        bull=0
        heavy=0
        if(l>=10000 or w>=10000 or h>=10000 or v>=1000000000):
            bull=1
        if(m>=100):
            heavy=1
        if(bull and heavy):
            return "Both"
        if(bull==0 and heavy==0):
            return "Neither"
        if(bull):
            return "Bulky"
        if(heavy):
            return "Heavy"
        """
        :type length: int
        :type width: int
        :type height: int
        :type mass: int
        :rtype: str
        """
        

File: my-code/2619-categorize-box-according-to-criteria/solution.cpp
class Solution {
public:
    string categorizeBox(int length, int width, int height, int mass) {
        
        long long l = length;
        long long w= width;
        long long h = height;

        long long volume = l*w*h;

        bool bull=false;
        bool hv = false;
        if(length>=1e4||width>=1e4||height>=1e4||volume>=1e9){
            //bulky:
            bull=true;
        }
        if(mass>=100)
        {
            //heavy;
            hv=true;
        }
        if(bull==true&&hv==true){
            return "Both";
        }
        if(bull==true&&hv==false){
            return "Bulky";
        }
         if(hv==false&&bull==false){
            return "Neither";
        }
        if(bull==false||hv==true){
            return "Heavy";
        }
        return "";
      
    }
};
